matchbook-sdk-rest/src/main/java/com/matchbook/sdk/rest/dtos/events/Market.java

```diff
@@ -16,9 +15,10 @@
     private MarketType marketType;
     private boolean inRunning;
     private boolean allowLiveBetting;
+    private Double handicap;
     private Double volume;
-    private Double backOverround;
-    private Double layOverround;
+    private BigDecimal backOverround;
```

Comment:
Why are you changing from Double to BigDecimal the overrounds?
Is it to match closely the implementation on MB? In theory if we are not manipulating those values, hence don't need a higher precision, Double can do fine, and are faster than BigDecimals.

Reply:
Good point. We should double-check the resources and consider using only `Double`s when not strictly necessary otherwise. I suppose I changed this to help the parser when I start working on it. I will revise the DTO's.

Reply:
I replaced a few `BigDecimal`s in 79051957e35d451f9e3b50690a4d4eb44731cb7d.